<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuTopologies">
    <property name="title">
     <string>Topologies</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComputations">
    <property name="title">
     <string>Computations</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTopologies"/>
   <addaction name="menuComputations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTopologyPIV">
   <property name="text">
    <string>TopologyPIV</string>
   </property>
  </action>
  <action name="actionTopologyPreproc">
   <property name="text">
    <string>TopologyPreproc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
